server/webpack.dev.config.js

```diff
@@ -8,6 +8,11 @@ module.exports = {
     'webpack/hot/only-dev-server',
     path.resolve(__dirname, '../client/src/index')
   ],
+  devServer: {
```

Comment:
(1) the host line was necessary for me. I couldn't view the server without it. The other lines are there to be explicit, though I believe those are the defaults for webpack-dev-server.